daemon/rpc/handlers.go

```diff
@@ -2,10 +2,67 @@ package rpc
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/40acres/40swap/daemon/crypto"
+	"github.com/40acres/40swap/daemon/database/models"
+	"github.com/40acres/40swap/daemon/lightning"
+	"github.com/40acres/40swap/daemon/swaps"
+	decodepay "github.com/nbd-wtf/ln-decodepay"
```

Comment:
can any pkg on LND rather than nbd can do this? LND should be more maintained and trusted

Reply:
done